app/models/compute_resources/foreman/model/openstack.rb

```diff
@@ -35,6 +36,22 @@ def tenant=(name)
       attrs[:tenant] = name
     end
 
+    def allow_ext_net?
+      !(attrs[:allow_ext_net] == 0)
+    end
+
+    def allow_ext_net
+      attrs[:allow_ext_net]
+    end
+
+    def allow_ext_net=(enabled)
+      if ['true', true, '1', 1].include?(enabled)
```

Comment:
I got a string each time for `enabled`, even when passing true/false for `checkbox_f`'s `checked_value` and `unchecked_value` parameters.